meson.build

```diff
@@ -1,14 +1,44 @@
-#
-# build script written by : Michael Brockus.
-# github repo author: Mike Karlesky, Mark VanderVoord, Greg Williams.
-#
-# license: MIT
-#
 project('unity', 'c',
-    license: 'MIT',
-    meson_version: '>=0.53.0',
-    default_options: ['werror=true', 'c_std=c11']
+  version: run_command(
+    find_program('python'), 'auto' / 'extract_version.py', 'src/unity.h',
```

Comment:
You don't need to find python here, because the extract_version.py script is runnable on its own as a regular program (it's executable and has a shebang). On Windows, Meson will scan the shebang and detect that it needs to be run with python.